packages/pipeline-editor/src/PipelineEditor/index.tsx

```diff
@@ -493,6 +493,15 @@ const PipelineEditor = forwardRef(
         }
 
         if (e.editType === "toggleOpenPanel") {
+          if (!panelOpen) {
```

Comment:
you need to add panelOpen to the dependency array

Reply:
hmmm, I though that would cause issues since we call setPanelOpen in this hook, though I could just remove this if block and let it check on close too even though it's silly

Reply:
no that should be okay since this is just a callback and not an effect